lib/platform-reactnative.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+require('setimmediate');
+require('fast-text-encoding');
+const crypto = require('isomorphic-webcrypto');
+
+exports.setImmediate = setImmediate;
+
+// Exported by 'isomorphic-webcrypto'
+exports.crypto = crypto;
+
+// Exported by 'fast-text-encoding'
+exports.TextEncoder = TextEncoder;
+
+// precompute byte to hex table
+const byteToHex = [];
+for(let n = 0; n <= 0xff; ++n) {
+  byteToHex.push(n.toString(16).padStart(2, '0'));
+}
+
+exports.bufferToHex = function bufferToHex(buffer) {
```

Comment:
Should consider moving this to another file that the browser file would also use.  platform-utils perhaps?